playbooks/openstack-deploy_network.yml

```diff
@@ -0,0 +1,28 @@
+---
+- hosts: localhost
+  gather_facts: no
+  connection: local
+  post_tasks:
+    - name: Load generic vars
+      include_vars: "{{ playbook_dir }}/../vars/common-vars.yml"
+    - name: Load openstack vars
+      include_vars: "{{ playbook_dir }}/../vars/deploy-on-openstack.yml"
+
+    - name: Create network
+      block:
+        - name: create network stack
+          register: network_stack_create_output
+          os_stack:
+            name: "{{ deploy_on_openstack_prefix }}"
+            template: "{{ playbook_dir }}/../heat-templates/openstack-network.yml"
+            parameters:
+              prefix: "{{ deploy_on_openstack_prefix }}"
+              network_external: "{{ deploy_on_openstack_external_network }}"
+
+        - name: show network stack output
+          debug:
+            var: network_stack_create_output
+
+        - name: Get network name
+          debug:
+            msg: "{{ (network_stack_create_output.stack.outputs | selectattr('output_key', 'search', 'network_name') | list | first).output_value }}"
```

Comment:
We should probably write the stack name out to a file like we do for ~/suse-socok8s-deploy/stackname. Then we can check if the stack already exists before creating it to make this more idempotent.
It would also be good to have a teardown function for this.

Reply:
Wouldn't the stackname (deploy_on_openstack_prefix) be enough to know what to delete? 

We can probably use the same approach as caasp for the deletion indeed.

Reply:
Both fixed I think. I wrote a file and integrated the cleanup in the teardown function.